### DISCUSSION TITLE
Script Winter of Code : Ask questions here

### DISCUSSION THREAD

**Discussion by ucalyptus** [ucalyptus/Spotify-Recommendation-Engine#82] [General]
Me or @forkbabu will reply.
Before you ask something , make sure you checkout the video of the app, issue lists, swoc slack channel and everything on the readme and other .md files in the repo. Thank You

**Comment by JoyclynUjunwaOgbonna** [comment]
HI Ucalyptus. How did you generate the token from spotify? 

I tried token = util.prompt_for_user_token(username,scope,cid,secret,redirect_uri). It took forever to run and didn't return any output.

**Comment by ucalyptus** [comment]
@JoyclynUjunwaOgbonna some parts of the process have changed . i longer maintain this project but will look to make a commit sometime in the next 12 months.
